corehq/blobs/management/commands/run_blob_export.py

```diff
@@ -18,7 +18,12 @@
 
 class Command(BaseCommand):
     """
-    Example: ./manage.py run_blob_export [options] export_domain_apps domain
+    Example: ./manage.py run_blob_export [options] domain
+
+    Dump XForms in parallel:
+        ./manage.py run_blob_export -e sql_xforms --liimit-to-db p0 domain
+         ...
+        ./manage.py run_blob_export -e sql_xforms --liimit-to-db pN domain
```

Comment:
So it should be okay to run this in parallel when running an export. Is it also possible to run them in parallel when running an import?